core/src/main/scala/kafka/utils/Json.scala

```diff
@@ -30,20 +32,11 @@ object Json {
   private val mapper = new ObjectMapper()
 
   /**
-   * Parse a JSON string into a JsonValue if possible. `None` is returned if `input` is not valid JSON.
-   */
+    * Parse a JSON string into a JsonValue if possible. `None` is returned if `input` is not valid JSON.
+    */
```

Comment:
Let's not change this indentation as it makes it inconsistent with the rest of the file.

Reply:
Ack.